src/WebJobs.Script.WebHost/Standby/StandbyManager.cs

```diff
@@ -66,8 +78,13 @@ public async Task SpecializeHostCoreAsync()
             // Trigger a configuration reload to pick up all current settings
             _configuration?.Reload();
 
+            string workerRuntime = _environment.GetEnvironmentVariable(LanguageWorkerConstants.FunctionWorkerRuntimeSettingName);
+            if (!string.IsNullOrEmpty(workerRuntime))
+            {
+                await _languageWorkerChannelManager.ReloadEnvironmentVariablesAsync(workerRuntime);
```

Comment:
I think you should roll these two calls into a single channelManager.Specialize(workerRuntime) call. That way you can remove the standby specific shutdown methods from the manager interface and internalize all the specialization logic in the manager. Then in the future if other operations are required for specialization, it all stays localized in the manager.

Reply:
done!